class_photo/bot.py

```diff
@@ -10,26 +10,34 @@
 bot = commands.Bot('-cp')
 
 def main():
-    load_dotenv()
     bot.run(os.getenv("TOKEN"))
 
 @bot.event
 async def on_ready():
     print('Bot Online!')
-    urls = await get_photos()
-    face.crop(urls)
+    await get_photos()
     await bot.logout()
```

Comment:
have you considered using bot.close() instead of bot.logout()? .logout() is deprecating